presto-main/src/main/java/com/facebook/presto/execution/TaskManagerConfig.java

```diff
@@ -41,6 +43,8 @@
         "task.level-absolute-priority"})
 public class TaskManagerConfig
 {
+    public static final Duration DEFAULT_INTERRUPT_RUNAWAY_SPLITS_TIMEOUT = new Duration(600, SECONDS);
```

Comment:
We don't usually have public defaults in config class. If we fix the constructor parameters for TaskExecutor, we can remove this default. In test, we usually hard code the default (which in this case is `new Duration(600, SECONDS)`) to make sure people are aware of the default changes.

Reply:
Removed and switched to instantiating the `TaskManagerConfig` like you suggested.